src/windows/input_hook.c

```diff
@@ -721,6 +723,60 @@ void CALLBACK win_hook_event_proc(HWINEVENTHOOK hook, DWORD event, HWND hWnd, LO
     }
 }
 
+static LRESULT CALLBACK WndProc(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
+{
+    switch (message)
+    {
+        case WM_DESTROY:
+            PostQuitMessage(0);
+            break;
+        case WM_DISPLAYCHANGE:
+            enumerate_displays();
+            break;
+        default:
+            return DefWindowProc(hwnd, message, wParam, lParam);
+    }
+    return 0;
+}
+
+static int create_invisible_window()
+{
+    WNDCLASSEX wcex = { 0 };
+    wcex.cbSize = sizeof(WNDCLASSEX);
+    wcex.style = WS_EX_NOACTIVATE;
+    wcex.lpfnWndProc = WndProc;
+    wcex.cbClsExtra = 0;
+    wcex.cbWndExtra = 0;
+    wcex.hInstance = hInst;
+    wcex.hIcon = NULL;
+    wcex.hCursor = NULL;
+    wcex.hbrBackground = NULL;
+    wcex.lpszMenuName = NULL;
+    wcex.lpszClassName = "Empty";
+    wcex.hIconSm = NULL;
+
+    if (!RegisterClassEx(&wcex)) return 0;
+
+    hWnd = CreateWindowEx(
```

Comment:
We should destroy the invisible window when the hook is stopped so that there's no memory leak.